scipy/stats/_axis_nan_policy.py

```diff
@@ -475,8 +476,7 @@ def hypotest_fun_out(*samples, **kwds):
                 # Addresses nan_policy == "raise"
                 contains_nans = []
                 for sample in samples:
-                    contains_nan, _ = (
-                        scipy.stats._stats_py._contains_nan(sample, nan_policy))
+                    contains_nan, _ = _contains_nan(sample, nan_policy)
```

Comment:
This is a a potentially good thing about moving the function - less trouble with circular imports.